### PR TITLE
Add admin level codes to all countries

### PR DESCRIPTION
**Pull request by ericboucher**
PR #974 made admin level codes mandatory for all layers.

@wadhwamatic @laurentS please have a look to make sure the selected admin codes are reasonable

### CODE REVIEW

**Comment by wadhwamatic** [comment]
@ericboucher - Would this also address https://github.com/WFP-VAM/prism-app/issues/604?

**Comment by wadhwamatic** [comment]
@ericboucher - I added a few changes. Looks good from my side

File: frontend/src/config/mongolia/layers.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "path": "data/mongolia/admin_boundaries.json",
     "opacity": 0.8,
     "admin_code": "NSO_CODE",
```
Comment (ericboucher) [inline frontend/src/config/mongolia/layers.json:6]: @wadhwamatic then this should likely be updated as well no?

Comment (laurentS) [reply frontend/src/config/mongolia/layers.json:6]: I took a guess when I copied `admin_code` into `admin_level_codes` here, but I was a little puzzled by the fact that the format of codes did not seem consistent with other countries (5 digits vs 6).

**Review by laurentS — APPROVED** [review]

**Pull request merged**